File: MSRsca_sie_vw/MSRsca_sie_vw.m
x1 = load('Sie9906_standLogRet.dat');
x2 = load('Vow9906_standLogRet.dat');

scatter(x1, x2, '.', 'b')
title('Log-Returns of Siemens versus Volkswagen')
xlabel('Standardized Log-Returns of Siemens')
ylabel('Standardized Log-Returns of Volkswagen')
xlim([-8 8])
ylim([-8 8])

File: MSRsca_sie_vw/MSRsca_sie_vw.R
# ---------------------------------------------------------------------
# Book:         
# ---------------------------------------------------------------------
# Quantlet:     MSRsca_sie_vw
# ---------------------------------------------------------------------
# Description:  MSRsca_sie_vw provides a scatterplot of daily 
#               standardized log-returns of Siemens versus Volkswagen.
# ---------------------------------------------------------------------
# Usage:        MSRsca_sie_vw
# ---------------------------------------------------------------------
# Inputs:       None
# ---------------------------------------------------------------------
# Output:       Scatterplot of daily standardized log-returns.
# ---------------------------------------------------------------------
# Example:     
# ---------------------------------------------------------------------
# Author:       Zografia Anastasiadou, 20110603
# ---------------------------------------------------------------------

graphics.off()
rm(list=ls(all=TRUE))
#setwd("C:/...")

x1 = read.table("Sie9906_standLogRet.dat")
x2 = read.table("Vow9906_standLogRet.dat") 

plot(x1$V1, x2$V1, col = "blue", xlab = "Standardized Log-Returns of Siemens", 
    ylab = "Standardized Log-Returns of Volkswagen", 
    main = "Log-Returns of Siemens versus Volkswagen", 
    xlim = c(-6, 6), ylim = c(-6, 6), pch = 20)